packages/@ourworldindata/grapher/src/core/Grapher.tsx

```diff
@@ -1965,11 +1965,34 @@ export class Grapher
     }
 
     @computed get showFacetControl(): boolean {
+        const {
+            hideFacetControl,
+            filledDimensions,
+            yColumnSlugs,
+            availableFacetStrategies,
+            isStackedArea,
+            isStackedBar,
+            isStackedDiscreteBar,
+            isLineChart,
+        } = this
+
+        if (hideFacetControl != undefined) return !hideFacetControl
+
+        // heuristic: if the chart doesn't make sense unfaceted, then it probably
+        // also makes sense to let the user switch between entity/metric facets
+        if (!availableFacetStrategies.includes(FacetStrategy.none)) return true
+
+        const showFacetControlChartType =
+            isStackedArea || isStackedBar || isStackedDiscreteBar || isLineChart
+
+        const hasProjection = filledDimensions.some(
+            (dim) => dim.display.isProjection
+        )
+
         return (
-            !this.hideFacetControl ||
-            // heuristic: if the chart doesn't make sense unfaceted, then it probably
-            // also makes sense to let the user switch between entity/metric facets
-            !this.availableFacetStrategies.includes(FacetStrategy.none)
+            showFacetControlChartType &&
+            !hasProjection &&
+            yColumnSlugs.length <= 2
```

Comment:
wait I'm confused about the `<= 2` here now, shouldn't it be `>= 2`? Or am I reading this wrong?

Reply:
Ahh, this one slipped through. I think it should be `showFacetControlChartType && (!hasProjection || yColumnSlugs.length <= 2)`

Reply:
(This is to make sure that faceting is enabled for charts like [this one](https://ourworldindata.org/grapher/population-past-future?facet=none) even though they have a projection.)

Reply:
Ah, right, makes sense then.